src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonParameterInfoOfT.cs

```diff
@@ -48,72 +38,5 @@ internal class JsonParameterInfo<T> : JsonParameterInfo
                 DefaultValue = TypedDefaultValue;
             }
         }
-
-        public override bool ReadJson(ref ReadStack state, ref Utf8JsonReader reader, out object? value)
```

Comment:
Note-to-self: the null handling done in these methods is handled by `JsonConverterOf<T>.TryRead` (which replaces the calls to these methods in the object converters).